frontend/src/pages/LogsPage/LogsPage.tsx

```diff
@@ -75,7 +75,7 @@ const LogsPage = () => {
 			},
 		},
 		skip: !project_id,
-		nextFetchPolicy: 'cache-first', // See: https://stackoverflow.com/a/66489126
+		fetchPolicy: 'cache-and-network',
```

Comment:
Prior to this change, when we updated the query input or set the time range, the request to fetch new logs wouldn't be run. 

There's not too much written about `cache-and-network` but it is what is recommended to use from the [docs](https://www.apollographql.com/docs/react/v2/caching/cache-interaction/#incremental-loading-fetchmore).

Reply:
I confirmed that forward scrolling works _and_ it's not making a double request (what the previous stackoverflow comment was addressing).